### ISSUE TITLE
Unexpected result when adding default to a base interface

### ISSUE THREAD

**Issue by miKerscher** [siom79/japicmp#276]
We have to following code in version 1.0.0:

```java
interface A {
}

interface B extends A {
}

abstract class BAbstract  implements B {
}

class BImpl extends BAbstract {
}
```
In version 1.0.1 we add the method foo with a default implementation to A:

```java
interface A {

   default void foo() {
       // ...
   }
}
```
japi-cmp produces the following incompatiblity:

JApiClass [fullyQualifiedName=BImpl, changeStatus=UNCHANGED, compatibilityChanges=[METHOD_ABSTRACT_ADDED_IN_IMPLEMENTED_INTERFACE]]

Expected is METHOD_DEFAULT_ADDED_IN_IMPLEMENTED_INTERFACE

The Problem does not occur when the version number is changed to 1.1.0

The bug can be produced in the github repo https://github.com/miKerscher/japi-cmp-repoduce-issue-276/
The master branch contains version 1.0.0
The branch dev-1.0.1 contains version 1.0.1

**Comment by garydgregory** [comment]
I've run into this many times as well, it is quite counterintuitive!

**Comment by siom79** [comment]
Added new compatibilty change ` METHOD_DEFAULT_ADDED_IN_IMPLEMENTED_INTERFACE`.

**Comment by miKerscher** [comment]
Thank you for your fast response.
When will the new version with the fix be available?

**Comment by siom79** [comment]
Released with 0.15.0
